embedded-google-storage/src/main/java/com/playtika/test/storage/StorageProperties.java

```diff
@@ -0,0 +1,31 @@
+package com.playtika.test.storage;
+
+import com.playtika.test.common.properties.CommonContainerProperties;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.Collection;
+
+import static java.util.Collections.emptyList;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ConfigurationProperties("embedded.google.storage")
+public class StorageProperties extends CommonContainerProperties {
+
+    public static final String BEAN_NAME_EMBEDDED_GOOGLE_STORAGE_SERVER = "embeddedGoogleStorageServer";
+    /** default port exposed by the fake-gcs-server */
+    static final int PORT = 4443;
+    /** project id for storage */
+    private String projectId = "my-project-id";
+    /** location for buckets */
+    private String bucketLocation = "US-CENTRAL1";
+    /** list with predefined buckets to be created */
+    private Collection<String> buckets = emptyList();
```

Comment:
Bucket creation operation supports additional parameters, that may be needed by someone in the future, so lets' replace this instead with:
```
private Collection<BucketProperties> buckets = emptyList();

static class BucketProperties {
 String name;
// other properties if needed can be added later
}
```